src/components/pages/tags/ShowPage.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { Button } from 'react-bootstrap';
+import { Link, useParams } from 'react-router-dom';
+import TagService from '@services/tagService';
+
+export default function ShowPage() {
+    const { id } = useParams();
+    const [tag, setTag] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        if (!id) return;
+        TagService.get(id)
+            .then(({data}) => {
+                setTag(data);
+                setLoading(false);
+            })
+            .catch(error => {
+                console.error('Error al cargar el tag:', error);
```

Comment:
When the task has already been deleted, the response returns a 401 error. This causes an error to appear in the console, the loading indicator to disappear, and no data to be shown on the page.
Let’s redirect the user to the /tag/ page in this case, so they’re not stuck on a broken screen.